docs/install/releases.md

```diff
@@ -7,12 +7,13 @@ We recommend enterprise customers test the compatibility of new releases with
 their infrastructure on a staging environment before upgrading a production
 deployment.
 
-We support two release channels:
-[mainline](https://github.com/coder/coder/releases/tag/v2.13.0) for the bleeding
-edge version of Coder and
-[stable](https://github.com/coder/coder/releases/latest) for those with lower
-tolerance for fault. We field our mainline releases publicly for one month
-before promoting them to stable.
+We support three releases of Coder concurrently. We release
+[mainline](https://github.com/coder/coder/releases/tag/v2.13.0) as the bleeding
```

Comment:
```suggestion
[mainline](https://github.com/coder/coder/releases/tag/v2.15.0) as the bleeding
```

per <https://github.com/coder/coder/releases/>

Reply:
good catch, we should make this update automatically

Reply:
fixed statically for now.